static/app/utils/replays/getPrevUserAction.tsx

```diff
@@ -2,29 +2,23 @@ import {relativeTimeInMs} from 'sentry/components/replays/utils';
 import {Crumb} from 'sentry/types/breadcrumbs';
 
 export function getPrevUserAction(
-  userActionCrumbs: Crumb[] | undefined,
+  userActionCrumbs: Crumb[],
   startTimestamp: number | undefined,
-  currentHoverTime: number | undefined
+  currentHoverTime: number
 ) {
-  if (
-    !startTimestamp ||
-    !userActionCrumbs ||
-    userActionCrumbs.length < 1 ||
-    currentHoverTime === undefined
-  ) {
+  if (!startTimestamp || userActionCrumbs.length < 1) {
```

Comment:
what happens if there is one crumb, and it's at the timestamp `0:00`? will it ever be highlights?